package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && npm run unit && npm run cover && npm run check-coverage && npm run snyk",
+    "test": "npm run lint && npm run unit && npm run cover && npm run check-coverage",
```

Comment:
I agree we need snyk or something similar. But I'm not sure this is the best place for it to live.
i.e. Should snyk really stop us from merging PRs like this one?

I think we need a better way of monitoring this stuff. Is there nothing like greenkeeper out there for security stuff?